website/content/en/preview/AWS/provisioning.md

```diff
@@ -1,216 +1,259 @@
----

```

Comment:
I'm not sure why the diff shows that all of this file was deleted and re-added, but looks like you just added the UserData section?

Reply:
I think it's line endings. I think this was originally using CRLF, and I might've switched it to LF. Let me fix that too.

Reply:
I think it's line endings. I think this was originally using CRLF, and I might've switched it to LF. Let me fix that too.